account_statement_base/views/account_bank_statement_line.xml

```diff
@@ -138,14 +138,14 @@
         <search>
             <field
                     name="payment_ref"
-                    filter_domain="['|', '|', ('payment_ref', 'ilike', self), ('ref', 'ilike', self), ('narration', 'ilike', self)]"
-                    string="Label, Ref or Notes"
+                    filter_domain="['|', '|', '|', ('amount', 'ilike', self), ('payment_ref', 'ilike', self), ('ref', 'ilike', self), ('narration', 'ilike', self)]"
```

Comment:
Operator `ilike` with a float number seems weird. How is it working?

Reply:
hi @pedrobaeza 
if I understood well, `ilike` can be use with a float thank to OCB/odoo/osv/operation.py line 1136 by casting to str